stream_alert/sink.py

```diff
@@ -43,29 +53,24 @@ def sink(self):
         group of alerts to the given SNS topic.
 
         Sends a message to SNS with the following JSON format:
-            default: 'default',
-            alerts: [
-                'rule_name': 'name',
-                'outputs': ['output1', 'output2'],
-                'payload': 'message_payload'
-            ]
+            {default: [
+                {
```

Comment:
this is actually an old alert format, can you update it to match https://github.com/airbnb/streamalert/blob/master/stream_alert/rules_engine.py#L209-L221